apps/src/templates/rubrics/LearningGoal.jsx

```diff
@@ -131,8 +132,7 @@ export default function LearningGoal({
           }
           if (json.understanding >= 0 && json.understanding !== null) {
             setDisplayUnderstanding(json.understanding);
-          } else {
```

Comment:
Why is this else removed?

Reply:
Realized it's unnecessary, since there shouldn't be any situation where you need to revert to invalidUnderstanding after an understanding has been selected.

Reply:
oh is it because it would already have `invalidUnderstanding` as the value?

Reply:
makes sense!

Reply:
exactly!